### PR TITLE
Nine new drink recipes

### PR DESCRIPTION
**Pull request by ShadyTheFirst**
## About the PR 
Adds three new drink recipes for bartenders to use - Blue Milk (50/50 Milk/Blueberry Juice), Laura Palmer (50/50 Coffee/Lemonade), and Romulale (25/25/50 Blueberry Juice/Blue Raspberry Juice/Vodka)

Edit: Also added Pine-Ginger (2 parts pineapple juice, 1 part ice, 1 part ginger ale), Vermont Breeze (4 parts lemonade, 1 part grenadine, 1 part tonic), Sail Boat (4 parts ginger ale, 1 part lime juice, 1 part ice), Catamount (4 parts ginger ale, 1 part orange juice, 1 part grenadine, 2 parts ice), Sun Tea (3 parts tea, 1 part orange juice, 1 part sugar), and Cafe Gele (6 parts fresh coffee, 1 part vanilla, 1 part sugar).

In-game image:

image
Left to right - Romulale, Blue Milk, Laura Palmer.

image
Left to right - Pine-Ginger, Vermont Breeze, Sail Boat, Catamount, Sun Tea, and Cafe Gele.

## Why's this needed? 
I think more drink options can be a pretty neat idea. Especially as 2/3 of the proposed recipes will involve some level of hydroponics to create.

Of course, the Blue Milk is a reference to the Star Wars drink - a little nod to a sci-fi classic.
The Laura Palmer is just a drink I personally like. 
The Romulale is inspired by Star Trek's Romulan Ale, a supposedly very strong alcoholic beverage. The recipe itself is referencing a little bit of trivia about the show, where the on-screen Romulan ale was partially Glacier (blue, berry-flavored) Gatorade.

## Changelog

```changelog
(+)Added nine new drink recipes, a mostly non-alcoholic.
```

### CODE REVIEW

**Comment by ShadyTheFirst** [comment]
> It's green.

I'm a little new to GitHub terminology - is this referencing something, an "all clear", or a comment on the color of the drinks? Thanks. 🙏

**Comment by TerraGS** [comment]
> > It's green.
> 
> I'm a little new to GitHub terminology - is this referencing something, an "all clear", or a comment on the color of the drinks? Thanks. 🙏

[Here's the reference](https://www.youtube.com/watch?v=YQAG0JEb718)

**Comment by ShadyTheFirst** [comment]
> > > It's green.
> > 
> > 
> > I'm a little new to GitHub terminology - is this referencing something, an "all clear", or a comment on the color of the drinks? Thanks. 🙏
> 
> [Here's the reference](https://www.youtube.com/watch?v=YQAG0JEb718)

😵‍💫 Went right over my head. Star Trek has surprisingly large number of alcoholic beverages.

File: code/modules/chemistry/Reagents-FoodDrink.dm [hunk 1]
```diff
@@ -1821,6 +1834,17 @@ datum
 			reagent_state = LIQUID
 			taste = "overwhelming"
 
+		fooddrink/alcoholic/romulale
+			name = "Romulale"
+			id = "romulale"
+			fluid_r = 20
+			fluid_g = 168
+			fluid_b = 254
+			alch_strength = 4
```
Comment (Tarmunora) [inline code/modules/chemistry/Reagents-FoodDrink.dm:1843]: way too high

**Review by Tarmunora — CHANGES_REQUESTED** [review]

File: code/modules/chemistry/Reagents-FoodDrink.dm (hunk 1 above)
Comment (ShadyTheFirst) [reply code/modules/chemistry/Reagents-FoodDrink.dm:1843]: I just looked through the other drinks and picked out a value one of the other ones had that was on the higher end. I'm not sure exactly how they scale, around what would be a suitable strength to still be a "strong" drink while not being too high?

Comment (Tarmunora) [reply code/modules/chemistry/Reagents-FoodDrink.dm:1843]: alch strength of 1 is pretty much the upper limit of 'mundane' drinks;  ~0.8 would be pretty strong

**Comment by ShadyTheFirst** [comment]
In hindsight I want to adjust the Romulale recipe to be more "ale" like. Reverting to a draft until I can adjust it.

**Comment by ShadyTheFirst** [comment]
> I have 6 drink recipes to propose if you want to add some more to the PR! Just ask and I'll tell - they're not references to media or anything they're just non-alcoholic iced drinks on the side of some novelty cup I have

Send them here and I'll add them to the PR.

**Comment by ShadyTheFirst** [comment]
> > Send them here and I'll add them to the PR.
> 
> Pine-Ginger
> 
> 8oz pineapple juice dash of ginger ale ice cubes cherry
> ....

I love all of these ideas. I'm not sure how many could be done without also adding new items - specifically the cinnamon, fruit slices, etc - Earl Grey irl is tea with bergamot orange oil, maybe some form of zesting fruit would have to be added to make that work, or make it tea+orange juice. The new items/mechanics could also be added to this PR but I'm not sure how complex the bartending experience is expected to be.

**Comment by ShadyTheFirst** [comment]
image
All suggested drinks have been added as possible creations, I'll update the PR in a bit. The one thing I'd like to add too might be ginger as a growable crop that can be brewed in a still to create ginger ale. Currently, ginger ale is nigh-unobtainable in most scenarios so a good chunk of drinks that require it are essentially off-limits - I asked in the discord if there was a reason for it being so hard to come by and it seems that most people agree that there's no real reason outside of nobody's done it yet (if there's a specific reason for ginger ale being tough to get lmk).

**Pull request merged**